src/racing/actors.h

```diff
@@ -72,38 +74,38 @@ void place_all_item_boxes(struct ActorSpawnData*);
 void init_kiwano_fruit(void);
 void destroy_all_actors(void);
 void place_course_actors(void);
```

Comment:
Can we rename this to:
`spawn_course_actors`?

In-fact replace `place_` with `spawn_` (we can do that in a different PR if you want)

If we want to show that these actors don't really move around the map (falling rocks move but the spawn point is probably static I dunno).

We could call these something like `spawn_static_course_actors`. I don't think static is quite suitable because that word should likely mean non-behaviour objects (that you can't run code on).

Reply:
I think just spawn are enough

Reply:
kk, sounds good to me.